### DISCUSSION TITLE
Please clarify the Licence Terms

### DISCUSSION THREAD

**Discussion by randy-armstrong** [mmistakes/so-simple-theme#342] [General]
The license requires:

> The above copyright notice and this permission notice shall be included in all
> copies or substantial portions of the Software.

1) Does using the theme on a website count as a "copy"?
2) What are you expecting in terms of credit when a website uses your theme?

**Comment by mmistakes** [comment]
I have no expectations of credit. The license is there because some of the libraries and such the theme uses have the same and require it.

**Comment by randy-armstrong** [comment]
Thx
